DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -101,6 +101,9 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
             if oldValue != areAutomaticUpdatesEnabled {
                 userDriver?.cancelAndDismissCurrentUpdate()
                 try? configureUpdater()
+#if !DEBUG
+                checkForUpdateIfNeeded()
```

Comment:
Triggers an "manual" update only when the user toggles between automatic/manual update. This bypasses the phased update group handling.